modules/swagger-codegen/src/main/resources/swift/api.mustache

```diff
@@ -16,14 +16,26 @@ extension {{projectName}}API {
 /** {{description}} */{{/description}}
 public class {{classname}}: APIBase {
 {{#operation}}
+    {{#allParams}}
+    {{#isEnum}}
+    /**
+
+    enum for parameter {{paramName}}
+    */
+    public enum {{{datatypeWithEnum}}}_{{operationId}}: String { {{#allowableValues}}{{#values}}
```

Comment:
This is solving the possible clash issue. But for people that don't have a clash, this is created a cumbersome API. Maybe we should add some logic Java-side to know if we have a conflict and do that only in that case?